compiler/MetadataUtils.ts

```diff
@@ -364,6 +364,9 @@ export default class MetadataUtils {
         // https://github.com/AdguardTeam/PopupBlocker/issues/238
         '*://online.bfgruppe.de/*',
         '*://*.online.bfgruppe.de/*'
```

Comment:
It seems that there is a missed comma `,`
The same in this line - https://github.com/AdguardTeam/PopupBlocker/pull/242/files#diff-2ab9c222fff241d52e1a592d1afbe9e017e794544870782a67933fe1e228434dR725

Reply:
damn, yes